rdr_service/genomic/genomic_job_components.py

```diff
@@ -796,10 +798,14 @@ def reconcile_metrics_to_genotyping_data(self):
             for file_type in self.genotyping_file_types:
                 if not getattr(metric, file_type[0]):
                     filename = f"{metric.chipwellbarcode}{file_type[1]}"
-                    file_exists = self._check_genotyping_file_exists(file.bucketName, filename)
-                    setattr(metric, file_type[0], file_exists)
+                    file_exists = self._get_full_filename(file.bucketName, filename)
+
+                    if file_exists != 0:
```

Comment:
Yes, I don't like how this is setup and will probably update it. Since other workflows use this method I needed to do some hacky changes. I'll fix in the next AW3 PR.